ui/components/MesheryPatterns.js

```diff
@@ -336,7 +336,7 @@ function MesheryPatterns({
   const [search, setSearch] = useState('');
   const [sortOrder, setSortOrder] = useState('');
   const [count, setCount] = useState(0);
-  const [pageSize, setPageSize] = useState();
+  const [pageSize, setPageSize] = useState(0);
```

Comment:
Thankyou @Adarsh-jaiss this should fix issue with undefined, I think we should have 10 as a default pagesize
```suggestion
  const [pageSize, setPageSize] = useState(10);
```